src/components/data-exploration/generic-gene-table/CreateCellSetModal.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {
+  Modal, Form, Button, Space, Select, InputNumber, Typography, Divider, Row, Alert,
+} from 'antd';// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
+
+const { Title } = Typography;
+
+const CreateCellSetModal = (props) => {
+  const { selectedGenes, onCancel } = props;
+  const [form] = Form.useForm();
+  const intialFormValues = selectedGenes.map((gene) => ({ selectedGenes: gene }));
+
+  const createCellSet = (filterForm) => {
+    console.log('filter form is ', filterForm);
+  };
+
+  const comparisonOptions = [{
+    value: 'greaterThan',
+    label: 'Greater than',
+  }, {
+    value: 'lessThan',
+    label: 'Less than',
+  },
+  ];
+
+  const formHeight = selectedGenes.length < 5 ? selectedGenes.length * 75 : 300;
+  return (
+    <Modal
+      visible
+      title='Create a new cell set based on gene expression'
+      onCancel={onCancel}
+      footer={null}
+      width='530px'
+    >
+      <Form
+        form={form}
+        onFinish={({ filterForm }) => createCellSet(filterForm)}
+        size='middle'
+      >
+        <Form.List
+          name='filterForm'
+          initialValue={intialFormValues}
+        >
+          {(fields) => (
+            <>
+              <Row style={{ overflow: 'scroll', height: formHeight }}>
```

Comment:
can you use `overflow-y:auto` instead, so the scroll bar will not appear if it doesn't need to appear?